decidim-dev/lib/decidim/dev/test/w3c_rspec_validators_overrides.rb

```diff
@@ -1,5 +1,39 @@
 # frozen_string_literal: true
 
+# This is a temporary fix to ignore some HTML/CSS validation issues with the
+# Decidim HTML validation process.
+#
+# See: https://github.com/decidim/decidim/pull/10014
+# Related: https://github.com/rails/rails/issues/46405
```

Comment:
Should we add the foundation URL too, so if/when that PR is merged we have it in the same place? 

```suggestion
# Related: 
# - https://github.com/rails/rails/issues/46405
# - https://github.com/foundation/foundation-sites/pull/12496
```